spec/models/article_spec.rb

```diff
@@ -137,3 +137,28 @@
     end
   end
 end
+
```

Comment:
So, do you not plan on comparing Feb and Mar, or just random 30 day periods (which still don't divide 365 easily)?

I think it makes sense to cover this before you end up with bad date logic.